### PR TITLE
Update system-requirements.md

### PR DESCRIPTION
**Pull request by leslielin-5**
adding "Stretched Cluster in AKSHCI and Aks on Windows Server" per request from @EkeleAsonye

### CODE REVIEW

File: azure-stack/aks-hci/system-requirements.md [hunk 1]
```diff
@@ -185,6 +185,10 @@ Download [URL allowlist (json)](https://raw.githubusercontent.com/MicrosoftDocs/
 
 ----
 
+#### Stretched Cluster in AKS on Azure Stack HCI and AKS on Windows Server
```
Comment (sethmanheim) [inline azure-stack/aks-hci/system-requirements.md:188]:
```suggestion
#### Stretched clusters in AKS on Azure Stack HCI and AKS on Windows Server 
```

File: azure-stack/aks-hci/system-requirements.md [hunk 2]
```diff
@@ -185,6 +185,10 @@ Download [URL allowlist (json)](https://raw.githubusercontent.com/MicrosoftDocs/
 
 ----
 
+#### Stretched Cluster in AKS on Azure Stack HCI and AKS on Windows Server 
+> [!NOTE]
+> At this time, stretched cluster has not been validated for different customer use cases and is not supported for use in your AKS on-premises environment. We advise that you use the backup and disaster recovery approach for your datacenter operational continuity.
```
Comment (sethmanheim) [inline azure-stack/aks-hci/system-requirements.md:190]:
```suggestion
> At this time, stretched clusters have not been validated for different customer use cases and is not supported for use in your AKS on-premises environment. We advise that you use the backup and disaster recovery approach for your datacenter operational continuity.  
```

**Comment by jborsecnik** [comment]
@sethmanheim, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by sethmanheim** [comment]
Hi @leslielin-5 I am closing this PR, as it's in the public repo. Since you're internal, can you please open a new one in the private repo? Thanks! #please-close

**Pull request closed without merging**